src/plugins/visualize/public/plugin.ts

```diff
@@ -61,6 +62,7 @@ export interface VisualizePluginStartDependencies {
   savedObjectsTaggingOss?: SavedObjectTaggingOssPluginStart;
   presentationUtil: PresentationUtilPluginStart;
   usageCollection?: UsageCollectionStart;
+  spaces: SpacesPluginStart;
```

Comment:
I remember you mention that, but is it impossible to make it an optional plugin? If it is possible, that's the preferred option (if plugin is disabled, visualizations will still load)

Reply:
Yep, it's optional plugin and I added `?` here